src/playercards/PlayerCardSpawner.ttslua

```diff
@@ -1,36 +1,73 @@
 Spawner = { }
 
--- Spawns a list of cards at the given position/rotation
+-- Spawns a list of cards at the given position/rotation.  This will separate cards by size -
+-- investigator, standard, and mini, spawning them in that order with larger cards on bottom.  If
+-- there are different types, the provided callback will be called once for each type as it spawns
+-- either a card or deck.
 -- @param cardList: A list of Player Card data structures (data/metadata)
 -- @param pos Position table where the cards should be spawned (global)
 -- @param rot Rotation table for the orientation of the spawned cards (global)
 -- @param sort Boolean, true if this list of cards should be sorted before spawning
--- @param callback Function, callback to be called after the card/deck spawns
+-- @param callback Function, callback to be called after the card/deck spawns.
 Spawner.spawnCards = function(cardList, pos, rot, sort, callback)
   if (sort) then
     table.sort(cardList, Spawner.cardComparator)
   end
+
+  local miniCards = { }
+  local standardCards = { }
+  local investigatorCards = { }
+
+  for _, card in ipairs(cardList) do
+    if (card.metadata.type == "Investigator") then
+      table.insert(investigatorCards, card)
+    elseif (card.metadata.type == "Minicard") then
+      table.insert(miniCards, card)
+    else
+      table.insert(standardCards, card)
+    end
+  end
+  -- Spawn each of the three types individually.  Each Y position shift accounts for the thickness
+  -- of the spawned deck
+  local position = { x = pos.x, y = pos.y, z = pos.z }
```

Comment:
Defensive copies are a hard habit to break.  But even if it's passed by value, mutating an input parameter is generally a bad idea, and it will bite you eventually :)